server/src/main/assets/stylesheets/main.less

```diff
@@ -224,8 +224,8 @@
 }
 
 .nav .staff-deployment-adjustment-container .container {
-  left: -19em;
-  top: 2.2em;
+//  left: -19em;
```

Comment:
do you wanna delete these rathe than comment them out?

Reply:
oops yeah